science-platform/templates/gafaelfawr-application.yaml

```diff
@@ -1,27 +1,27 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: auth
+  name: gafaelfawr
```

Comment:
I'm already excited to misspell this every time until I learn to spell this new wonderful word.